foxy/distribution.yaml

```diff
@@ -6221,6 +6221,11 @@ repositories:
       url: https://github.com/ros2/rviz.git
       version: foxy
     status: maintained
+  rviz_static_ground_image:
```

Comment:
So there are 3 different names I'm seeing:

The key you are using here is `rviz_static_ground_image`.  The repository name is `rviz-ground-image`.  And the package name in the [package.xml](https://github.com/Plaba/rviz-ground-image/blob/main/package.xml) is `ground_image`.

Let's make all of these the same, preferably with an `rviz` prefix.

Reply:
Thanks for pointing that out! I've renamed everything to `rviz_ground_image`.